src/gfx/lib.rs

```diff
@@ -62,6 +62,20 @@ pub struct Graphics<D: device::Device> {
     context: batch::Context<D::Resources>,
 }
 
+impl<D: device::Device> std::ops::Deref for Graphics<D> {
+    type Target = batch::Context<D::Resources>;
+
+    fn deref(&self) -> &batch::Context<D::Resources> {
+        &self.context
+    }
+}
+
+impl<D: device::Device> std::ops::DerefMut for Graphics<D> {
+    fn deref_mut(&mut self) -> &mut batch::Context<D::Resources> {
+        &mut self.context
+    }
+}
+
```

Comment:
This allows `make_batch`, `make_core`, and `bind` to be called from `Graphics`, instead of wrapping them up into pass-through methods.